pom.xml

```diff
@@ -415,10 +437,20 @@
             <artifactId>jackson2-geojson</artifactId>
             <version>0.8</version>
         </dependency>
+        <dependency>
+            <groupId>org.mapdb</groupId>
+            <artifactId>elsa</artifactId>
+            <version>3.0.0-M5</version>
+        </dependency>
         <dependency>
             <groupId>com.axiomalaska</groupId>
             <artifactId>polyline-encoder</artifactId>
             <version>0.1</version>
         </dependency>
+        <dependency>
+          <groupId>com.google.cloud</groupId>
+          <artifactId>google-cloud-storage</artifactId>

```

Comment:
What is Google cloud storage being used for? I don't see any references to it in the diff.